resources/views/internet/internet_access/internet_access.blade.php

```diff
@@ -0,0 +1,14 @@
+<div class="card">
+    <div class="card-header">@lang('internet.internet_access')</div>
+    <div class="card-body">
+        @if($internet_access->has_internet_until > \Carbon\Carbon::now())
+            <div class="alert alert-success">
+                <p>@lang('internet.has_internet', ['ends' => $internet_access->has_internet_until])</p>
```

Comment:
Missing row for `internet.has_internet` in HUN translations.
It would be reasonable to add that since the creation of that requires this line. Should contain a placeholder named `ends` in that.

Reply:
It is there.
resources/lang/hu/internet.php:36